opendistro-for-elasticsearch/hooks/run

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+exec 2>&1
+
+# Don't use the default password, ok?
+if [ "{{cfg.securityconfig.internal_users.admin.hash}}" = '$2a$12$VcCDgh2NDk07JGN0rjGbM.Ad41qVR/YFJcgHp0UGns5JDymv..TOG' ]; then
+    echo "=================================================="
+    echo "WARNING: UNSAFE DEFAULT PASSWORD FOR ADMIN USER!!!"
+    echo "WARNING: USE FOR TESTING PURPOSES ONLY!!!"
+    echo "WARNING: UNSAFE DEFAULT PASSWORD FOR ADMIN USER!!!"
+    echo "=================================================="
+fi
+
+free_mem="$(free -m | grep Mem | awk '{print $2}')"
```

Comment:
If we do decide to keep this memory calculation logic, what dependency is providing the `free` command? I believe its in `busybox-static` which is only included by the test suite.

Reply:
I think having a simple memory check here is pretty essential to ensuring a working cluster. I'll add the dependency. 👍